doc/code-review.md

```diff
@@ -0,0 +1,20 @@
+## Code Review
+
+For all our projects of the [Thoth Station](http://thoth-station.ninja/) we require code reviews before merging the code. We use [Zuul](https://zuul-ci.org/) to implement [project gating](https://zuul-ci.org/docs/zuul/user/gating.html).
+
+Code is merged by zuul, but before a pull request is merged, zuul runs all project specific jobs of the so called 'gate pipeline'. If all jobs are successful, code is merged. If a job fails, the pull request will get a comment added and the pipeline finishs without merging.
+
+The requirements and triggers for the gate pipeline are defined by [zuul-config's gate pipeline](https://github.com/thoth-station/zuul-test-config/blob/master/zuul.d/_pipelines.yaml#L45-L55):
+
+### Requirements
+
+The following requirements must be met to start merging a Pull Request:
+
+* a 'approved' [GitHub review](https://help.github.com/articles/about-pull-request-reviews/) must exists
+* the 'approved' [label](https://help.github.com/articles/about-labels/) must be set
```

Comment:
I guess this one is not needed anymore.

Reply:
so how do we want the process to be? just an approved label? just a approved review?

if we go with the approved review, we need all auto update PRs to be approved...